WcaOnRails/app/controllers/users_controller.rb

```diff
@@ -69,13 +69,13 @@ def select_nearby_delegate
 
   def edit_avatar_thumbnail
     @user = user_to_edit
-    return if redirect_if_cannot_edit_user(@user)
+    return if redirect_unless_user(:can_change_users_avatar?, @user)
```

Comment:
These `return if ... unless ...` constructs really confuse me. What do they mean?

Especially at the last line of a function, we will return regardless.